### PR TITLE
Text alignment

### PR DESCRIPTION
**Pull request by Jupeyy**
i think the round borders create a bit of optical illusion, e.g. the "V" in edtior is only 1 pixel of, but looks like its completly left xd

i checked at few resolutions with gimp and only saw around 1 pixel margin of error, but go ahead and test yourself :D

### CODE REVIEW

File: src/game/editor/editor.cpp [hunk 1]
```diff
@@ -664,11 +664,11 @@ int CEditor::DoButton_Tab(const void *pID, const char *pText, int Checked, const
 	return DoButton_Editor_Common(pID, pText, Checked, pRect, Flags, pToolTip);
 }
 
-int CEditor::DoButton_Ex(const void *pID, const char *pText, int Checked, const CUIRect *pRect, int Flags, const char *pToolTip, int Corners, float FontSize)
+int CEditor::DoButton_Ex(const void *pID, const char *pText, int Checked, const CUIRect *pRect, int Flags, const char *pToolTip, int Corners, float FontSize, bool Align)
 {
 	RenderTools()->DrawUIRect(pRect, GetButtonColor(pID, Checked), Corners, 3.0f);
 	CUIRect NewRect = *pRect;
-	UI()->DoLabel(&NewRect, pText, FontSize, 0, -1);
+	UI()->DoLabel(&NewRect, pText, FontSize, 0, -1, Align ? 0 : 1);
```
Comment (def-) [inline src/game/editor/editor.cpp:671]: Is this correct? If Align is set then don't align vertically?

Comment (Jupeyy) [reply src/game/editor/editor.cpp:671]: i made it a bit like hori vert, so 1 means its alinged to the bottom(default) and 0 is centering and -1 is not implemented yet

**Comment by Jupeyy** [comment]
oh forgot the editor close, is there anymore?

**Comment by def-** [comment]
Not sure, just saw that one.

File: src/game/editor/editor.cpp (hunk 1 above)
Comment (def-) [reply src/game/editor/editor.cpp:671]: But then Align should be an int, right?

Comment (Jupeyy) [reply src/game/editor/editor.cpp]: fixed

**Pull request merged**